ldms/src/contrib/store/timescale/LICENSE.txt

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
@tom95858 @wooloo1121 it would be helpful for us at Sandia who have to periodically conduct copyright clearances if there is one file with just the license (LICENSE.txt is an ok name), as well as being prepended to the source files. The license file should state in some way that it covers the timescaledb plugin sources only.

Reply:
@baallan @tom95858 So it is kept for now and I added one line to the file stating that it covers the timescaledb store plugin only. I'm not sure if it is the right way to do it, please let me know I did something wrong. And by "prepended to the source files", do you @baallan mean I should add the whole license file to the top of the source code file? I'm a little confused cause @tom95858 told me to add some other text regarding copyright to the top of the source file.

Reply:
@baallan, @wooloo1121: the license in LICENSE.txt is wrong. The code is _not_ licensed under the Apache license terms. Please remove this file.

Reply:
@tom95858 ok it is removed.

Reply:
Thank you!